ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -28,82 +36,101 @@ type ReduxMappedProps = Readonly<{
 
 type Props = ReduxMappedProps & NavigationInjectedProps;
 
+type State = Readonly<{
+  isTransactionStarted: boolean; // it will be true when the user accepted to proceed with a transaction and he is going to display the datail of the transaction as receipt
```

Comment:
typo (datail -> detail) also better use a multiline comment before the property instead of a very long comment on a single line